content/events/hacktoberfest-2020/index-md.md

```diff
@@ -0,0 +1,46 @@
+---
+meta_desc: Join Pulumi for Hacktoberfest and help us build open source integrations, automation and libraries.
```

Comment:
This is causing the build to fail b/c the linter expects this to be <160 characters

Reply:
addressed, now 99 characters.